core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -417,13 +424,15 @@ public String toString() {
               String.format("Comma-separated search path for %s.", Constants.SITE_PROPERTIES))
           .setIgnoredSiteProperty(true)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.ALL)
           .build();
   public static final PropertyKey TEST_MODE =
       new Builder(Name.TEST_MODE)
           .setDefaultValue(false)
           .setDescription("Flag used only during tests to allow special behavior.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setIsHidden(true)
+          .setScope(Scope.ALL)
```

Comment:
Should this be `Scope.NONE`

Reply:
One problem now is that there is no fix criteria for which scope a property falls in. My criteria is that if the PropertyKey is used in master or worker only, then they are Scope.MASTER or Scope.WORKER. If a property is used by both master and worker, or job master and job worker, then it falls to Scope.Server. If it's used by both server and client, it falls to ALL. For some property key that ony used by certain integration like MESOS, it falls to Scope.NONE. TEST_MODE is used in server and common codes.

Reply:
I see. It sounds like we need to have more defined criteria for `Scope.NONE`. I am fine making this `ALL` for now.